test/ocamlorg_web/graphql_test.ml

```diff
@@ -0,0 +1,201 @@
+module Package = Ocamlorg.Package
+
+type info = OpamPackage.Name.t * string option
+
+type package_info = info list
+
+type info_result =
+  { name : string
+  ; constraints : string option
+  }
+
+type packages_result =
+  { total_packages : int
+  ; packages : Package.t list
+  }
```

Comment:
What do you need these types for?

Reply:
Wow, turns out they were not necessary. I added them to match the types with graphql.ml. It's been removed, thank you